ios/Exponent/Kernel/Views/Loading/EXManagedAppSplashScreenController.m

```diff
@@ -0,0 +1,92 @@
+//
+//  EXManagedAppSplashScreenController.m
+//  Expo Go (unversioned)
+//
+//  Created by andrew on 2021-06-07.
+//  Copyright © 2021 650 Industries. All rights reserved.
+//
+
+#import "EXManagedAppSplashScreenController.h"
+#import "MBProgressHUD.h"
+#import "EXSplashScreenHUDButton.h"
+
+
+@interface EXManagedAppSplashScreenController()
+
+@property (nonatomic, weak) NSTimer *warningTimer;
+@property (nonatomic, weak) MBProgressHUD *warningHud;
+@property (nonatomic, strong) UIView *splashScreenView;
```

Comment:
From what I see the base class `SplashScreenController` is already holding a strong reference to this view 🤔 Not sure about `strong` modifier here 🧐 
Unfortunately, AFAIK, there's no `protected` property in Objective-C to get rid of this altogether 😞